code/parachain/runtime/picasso/src/gates.rs

```diff
@@ -85,8 +84,8 @@ impl Contains<RuntimeCall> for BaseCallFilter {
 
 impl call_filter::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type EnableOrigin = EnsureRootOrHalfNativeTechnical;
-	type DisableOrigin = EnsureRootOrOneThirdNativeTechnical;
+	type EnableOrigin = EnsureRoot<Self::AccountId>;
+	type DisableOrigin = EnsureRoot<Self::AccountId>;
```

Comment:
this must be uber fast. 

it must not allow to block sudo and referenda calls so I guess

Reply:
if we do whitlist, does it count as uber fast. i can keep technical commitee. for not allowing to block, i can add a hook